about.html

```diff
@@ -7,12 +7,137 @@
+        </div>
+      </div>
+    </div>
+  </header>
+  <nav>
+    <div class="container-fluid position-fixed nav-container">
+      <div class="hamburger">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+      <div class="row align-items-center">
+        <div class="col-12 col-md-5 d-flex justify-content-center align-items-center pt-md-0 pb-md-0">
+          <a href="index.html" class="d-flex justify-content-center" ><img src="img/logo.png" class="nav-icon pt-md-0"></a>
+        </div>
+        <div class="col col-md-4 d-flex align-items-center fw-bold">
+          <ul class="nav-links mb-0">
+            <a href="about.html"><li>About</li></a>
+            <a href=""><li>Program</li></a>
+            <a href=""><li>Join</li></a>
+            <a href=""><li>Sponsor</li></a>
+            <a href=""><li>News</li></a>
+          </ul>
+        </div>
+        <div class="col-md-3 align-items-center justify-content-start d-none d-md-flex ps-5">
+          <p class="p-1 m-1 campaign-text">CC Campaign</p>
+        </div>
+      </div>
+    </div>
+  </nav>
+  <section>
+    <div class="container-fluid p-0 intro-bg-layer">
+      <div class="container-fluid" id="intro">
+        <div class="row justify-content-center pt-md-5 mt-md-5">
+          <div class="col-12 col-md-8 p-0 d-flex pt-md-5 flex-column align-items-center">
+            <h1 class="meeting-count text-center fs-4">GET READY FOR SUMMIT 2023!</h1>
+            <h1 class="fs-5 mb-1 intro-hello pt-4 text-center">"GET READY TO CHANGE YOUR LIVE!"</h1>
+            <h1 class="intro-title mb-4 text-center pt-3 pb-3 pe-md-5 ps-md-5">SUPREME SUMMIT TO CHANGE YOUR LIFE 2023</h1>
+            <div class="orange-line pt-md-2"></div>
+            <p class="p-4 pt-4 pb-4 mb-0 mt-3 text-center intro-desc pe-md-5 ps-md-5 ms-md-3 me-md-3"><strong>What exactly makes this event so unique and amazing?</strong><br><br> Supreme summit was conceived in us as the quest to become the best version of yourself and serve the world with your greatness. We want to bring everyone into the same space to create together. This event is designed to not only teach, inspire, and empower you, but to let everyone come together with people who are on the same journey and lift each other up. <br><br> <strong>THAT’S what this Summit offers - come join us!</strong></p>
+            <div class="d-flex align-items-center flex-column pt-5 pb-5 contact-bg">
+              <p class="intro-contact mb-0 text-center ps-4 pe-4">Please contact us via Email for more further questions about Supreme Summit 2023!</p>
+              <p class="intro-email mb-0">supremesummit2023@myanmar.org</p>
+            </div>
+          </div>
+        </div>
+      </div> 
+    </div>
+  </section>
+  <section>
+    <div class="container-fluid">
+      <div class="row row-cols-md-1 justify-content-md-center logo-sec pb-4 pt-md-5 pb-md-5 mb-md-3">
```

Comment:
- [optional] please all the red text make the section kind of different from the original design. please remember following the design guidelines and make it a habit reproducing the same thing.

yours
![Screenshot (1073)_LI](https://github.com/K0ppai/first-capstone-project/assets/65298048/5588bebc-93bf-4a32-a141-058908e04858)
original
![Screenshot (1074)](https://github.com/K0ppai/first-capstone-project/assets/65298048/7d964738-c29f-4a0c-a98d-ab5d41c2830d)